scripts/devenv-builder/configure-vm.sh

```diff
@@ -102,6 +103,15 @@ baseurl=https://mirror.openshift.com/pub/openshift-v4/\$basearch/dependencies/rp
 enabled=1
 gpgcheck=0
 skip_if_unavailable=0
+EOF
+
+    sudo tee "/etc/yum.repos.d/${FAST_DATAPATH_REPO_NAME}.repo" >/dev/null <<EOF
+[${FAST_DATAPATH_REPO_NAME}]
+name=fast datapath for RHEL ${OSVERSION}
+baseurl=https://rhsm-pulp.corp.redhat.com/content/dist/layered/rhel9/$(uname -i)/fast-datapath/os/
```

Comment:
This is an internal URL, so we cannot use it.

Reply:
We may want to ask the ART team to include the v3 package alongside the v2.
However, it's still not clear to me where we expect getting the v3 package in production.